lang/src/main/java/org/kaazing/k3po/lang/internal/ast/AstConnectNode.java

```diff
@@ -93,4 +94,20 @@ public String getBarrier() {
     public void setBarrier(String barrier) {
         this.barrier = barrier;
     }
+
+    public AstValue getLocation() {
+        return location;
+    }
+
+    public void setLocation(AstValue location) {
```

Comment:
What was the purpose of having `AstLocation` super type of `AstLocationLiteral`?
I thought it was to have a common super type for both `AstLocationLiteral` and `AstLocationExpression` which is constrained to return type of `URI, but perhaps I am not remembering accurately.